HISTORY.rst

```diff
@@ -10,7 +10,15 @@ New features and enhancements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * Adjustment methods of `SBCK <https://github.com/yrobink/SBCK>`_ are wrapped into xclim when that package is installed. (:issue:`1109`, :pull:`1115`).
     - Wrapped SBCK tests are also properly run in the tox testing ensemble. (:pull:`1119`).
-* New indices for droughts: SPI (standardized precipitations) and SPEI (standardized water budgets) (:issue:`131`, :pull:`1096`)
+* Method ``FAO_PM98`` (based on Penman-Monteith formula) to compute potential evapotranspiration. (:pull:`1122`).
+
+New indicators
+^^^^^^^^^^^^^^
+* Drought-related indicators: SPI (standardized precipitations) and SPEI (standardized water budgets). (:issue:`131`, :pull:`1096`).
+
+Breaking changes
+^^^^^^^^^^^^^^^^
+* ``water_budget`` and ``potential_evapotranspiration`` now accept radiation (rlds, rlus, rsds, rsus), wind (u2), and relative humidity (hurs) as optional parameters. (:pull:`1122`).
```

Comment:
```suggestion
* ``water_budget`` and ``potential_evapotranspiration`` now accept radiation (rlds, rlus, rsds, rsus), wind (sfcWind), and relative humidity (hurs) as optional parameters. (:pull:`1122`).
```
The cmip6 name, or at least the one we've being using, for wind speed is `sfcwind`. And as far as I know, it doesn't prescribe a specific height?

I'll let you change it everywhere, make suggestions for each case would be quite chaotic ;).

Reply:
I think I changed in the code everywhere, forgot the docs it seems. I'll check everywhere thanks.